### DISCUSSION TITLE
new item in  tree.ini

### DISCUSSION THREAD

**Discussion by abyrnoradan** [WinEdt-Team/WinEdt-Support#62] [1.1 General Help and Discussion]
I wrote the following to define a "subpart" item

  ITEM="\parts{?}"
    MODE_FILTER="TeX"
    CASE_SENSITIVE=1
    BEGINNING_OF_LINE_ONLY=1
    CURRENT_DOCUMENT_ONLY=1
    ICON="Part"
    LEVEL=5
    CAPTION="%?"
    BALANCED="{}\"
    ON_CLICK_MACRO="TreeTrack(2,1);"
    ON_DBL_CLICK_MACRO="TreeTrack(2,2);"
END="\parts{?}"  

I put level=5 because it's subpart and i wanted this "parts" to be indent but ...

of course ... if I'm here ... it doesn't work and i have no idea why 
I just copy and paste a part of the "part" item

What is missing ?

**Comment by Alex-Simonic** [comment]
Perhaps you did not copy enough. I added the following two lines:

```
    ALL_OPENED_DOCUMENTS=0
    COMPLETE_PROJECT_TREE=1
```

and it works as expected. If you are using the main file `COMPLETE_PROJECT_TREE=1` is a must. The Help explains the meaning of all available options. I tested with the following lines in the main file:

```
\part{part 1}
\parts{subpart 1}
\chapter{ch1}
\parts{subpart 2}
\chapter{ch1}

\part{part 2}
\parts{subpart 1}
\chapter{ch1}
\parts{subpart 2}
\chapter{ch1}

```
 TOC in the tree reflected the structure as intended (with subparts inside parts).

**Comment by abyrnoradan** [comment]
Thanks a lot !

Indeed  I just didn't  copy enough.

Now "it   waarks !"
